muni_portal/settings.py

```diff
@@ -75,7 +75,6 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
-    "wagtail.core.middleware.SiteMiddleware",
```

Comment:
Why are you deleting this?

Reply:
It's deprecated.
![Selection_140](https://user-images.githubusercontent.com/338943/97857820-556f3b00-1d0f-11eb-9bf9-caa8b18c9f8f.png)